.changeset/quiet-suits-count.md

```diff
@@ -0,0 +1,55 @@
+---
+'@ithaka/pharos': major
+'@ithaka/pharos-cli': minor
+'@ithaka/pharos-site': minor
+---
+
+require manual component registration:
+
+* Remove component self-registration
+* Scope registries for components composed of other Pharos components using [scoped-registry-mixin](https://github.com/lit/lit/tree/main/packages/labs/scoped-registry-mixin)
+* Update React wrapper to detect defined custom tag name
+* Create `PharosElement` base class which sets a custom attribute `data-pharos-component` as a common identifier
+* Set components to extend `PharosElement`
+* Update documentation
+
+**Migration Guidelines**
+
+1. Remove all individual imports of Pharos web components.
+
+2. Register them with a custom tag in your app's entrypoint:
+
+    ```javascript
+    import { PharosAlert } from '@ithaka/pharos/lib/components/alert/pharos-alert';
+
+    customElements.define('pharos-alert-homepage', PharosAlert);
+    ```
+
+2. Add the [scoped custom element registry polyfill](https://github.com/webcomponents/polyfills/tree/master/packages/scoped-custom-element-registry) to your app.
```

Comment:
Include a note to be careful to avoid double-ups (e.g. if it's already added in a more global context)?